content/docs/command-reference/move.md

```diff
@@ -69,6 +71,46 @@ outs:
     path: other.csv
 ```
 
+### Manually renaming stage outputs
+
+In this example, let's rename stage output `keras.h5` to `model.h5`:
+
+```dvc
+$ cat dvc.yaml
+stages:
+  train:
+    cmd: python train.py
+    ...
+    outs:
+    - keras.h5
+```
+
+First we need to change the name of model file in code (`train.py`). Next we
```

Comment:
grammar, grammar

Reply:
But maybe just skip the whole `cmd:` like to simplify this explanation and avoid distractions.